mlflow/fastai.py

```diff
@@ -201,6 +202,7 @@ def log_model(
                           base64-encoded.
 
     :param kwargs: kwargs to pass to `fastai.Learner.export`_ method.
+    :param await_creation: Wait until the associated model version is finished being created
```

Comment:
should probably mention the max wait time

Reply:
Done